profiles/Pranay202.json

```diff
@@ -0,0 +1,6 @@
+{
+    "image": "https://user-images.githubusercontent.com/68046838/172583862-aab04eeb-d77f-42b3-8678-b05a23daf57e.png",
+    "issueId": 1161,
+    "name": "Pranay Kharabe",
+    "username": "Pranay202"
```

Comment:
Oh well, now that I noticed, it looks like there's an indentation issue here.
```suggestion
  "image": "https://user-images.githubusercontent.com/68046838/172583862-aab04eeb-d77f-42b3-8678-b05a23daf57e.png",
  "issueId": 1161,
  "name": "Pranay Kharabe",
  "username": "Pranay202"
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

Reply:
I have updated it according to the suggested changes. Please review @Panquesito7